.gitignore

```diff
@@ -269,3 +269,14 @@ binaries/
 
 # Do not include launch settings
 launchSettings.json
+
+.dotnet/
+artifacts/
+/.packages
+eng/common/cross/
+eng/common/internal/
+eng/common/native/
+eng/common/performance/
+eng/common/post-build/
+eng/common/sdl/
+eng/common/templates/
```

Comment:
Are you sure you don't need templates? I would imagine that those plus the post-build are important, so you probably want to remove those from here.